### DISCUSSION TITLE
안녕하세요? xbrl  회사별  비교 분석하려고 합니다.

### DISCUSSION THREAD

**Discussion by CHOKYUNGCHO** [josw123/dart-fss#174] [General]
비교 분석을 하려고 하는데 많이 막힙니다. 개발 방향이 같다면 같이 개발을 진행하고 싶습니다.현재 라이브러리의 개발 방향,진행 방법 등을 이야기하고 싶습니다. 감사합니다.

**Comment by josw123** [comment]
통합 분석을 위해서는 아마 계정에 명이나 concept-id 등을 비교하시면 될 것으로 생각됩니다. 그러나 일부 계정의 경우 명칭이나 concept-id가 일관되지 않으므로 이를 처리하면 아마 비교 분석이 가능하실 것으로 생각됩니다
